### DISCUSSION TITLE
1ST MILESTONE COMPLETE!!! User accounts

### DISCUSSION THREAD

**Discussion by EnZon3** [EnZon3/Drop#6] [Announcements]
Hello y'all!

I have finished the first major milestone which is.. User accounts!
Profiles aren't done yet but they should be done soon.

Check it out @ https://drop-beta.vercel.app

**Comment by MilesWK** [comment]
Whoa! The CSS is great! However, once you allow github access, it takes you to [http://localhost:3000/?code=0acd79fa-3525-472a-a4ea-7484688ff593](http://localhost:3000/?code=0acd79fa-3525-472a-a4ea-7484688ff593) and then you cannot do anything. Is this part of the "Profiles aren't done yet?

Also, and I don't know if you can change this but: the favicon is the swelve logo

**Reply by EnZon3** [reply]
Yep, it's the "profiles aren't done yet" part. I haven't figured out a good favicon yet, or a logo quite frankly. Any good suggestions for a logo would be appreciated!

**Reply by EnZon3** [reply]
OH WAIT

**Reply by EnZon3** [reply]
ill fix it 1sec

**Reply by EnZon3** [reply]
I actually have no clue why that's happening

**Comment by MilesWK** [comment]
You are an awesome developer. You'll figure it out!

**Reply by EnZon3** [reply]
Fixed

**Reply by EnZon3** [reply]
i think

**Reply by EnZon3** [reply]
Yep, fixed

**Comment by EnZon3** [comment]
Upd: if you didn't see, I have fixed user authentication flow

**Reply by MilesWK** [reply]
Looks great! Good work!
